modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -1,10 +1,15 @@
 (ns metabase.driver.redshift-test
-  (:require [clojure.test :refer :all]
+  (:require [clojure.string :as str]
+            [clojure.test :refer :all]
+            [metabase.driver.sql-jdbc.execute :as execute]
             [metabase.plugins.jdbc-proxy :as jdbc-proxy]
-            [metabase.test
-             [fixtures :as fixtures]
-             [util :as tu]]
-            [metabase.test.data.datasets :refer [expect-with-driver]]))
+            [metabase.query-processor :as qp]
+            [metabase.test :as mt]
+            [metabase.test.data.datasets :refer [expect-with-driver]]
```

Comment:
you've probably found this out already but the linters are going to complain if you import namespaces and then don't use them

Reply:
Another quick heads up:

This is related to the switch from uppercase param-names to backticks in docstrings: we switched from using the [Expectations](https://github.com/clojure-expectations/expectations) library for testing to `clojure.test` last year mostly because of better support in 3rd-party tools. I wrote [a handful of macros](https://github.com/metabase/metabase/blob/master/test/expectations.clj) that expand Expectations-style `expect` tests into `clojure.test` `deftest` ones so we could switch without having to rewrite thousands of tests immediately. 

As with the param names, we've been reworking old tests so they use `deftest` rather than the legacy `expect` in a piecemeal fashion. I usually rework a namespace whenever I add tests to one if it's not already reworked. If you feel like reworking any tests, please feel free to do so. Just make sure to take the time and rework the tests thoughtfully so they follow the spirit of `clojure.test` rather than just macroexpanding the `expect` forms and programatically generating names for them. There are a lot of cases where we can make test code a lot more concise with `clojure.test` so it's worth taking the time to do so.

Either way, whatever you do, don't add any new tests that use `expect` or any of the various other versions we have out in the wild